pombola/core/management/commands/core_sync_everypolitician_uuid.py

```diff
@@ -0,0 +1,54 @@
+from everypolitician import EveryPolitician
+
+from django.core.management.base import BaseCommand
+
+from pombola.core.models import Person
+
+
+class Command(BaseCommand):
+    help = 'Sync EveryPolitician UUID to Person.everypolitician_uuid field'
+
+    def add_arguments(self, parser):
+        parser.add_argument('everypolitician_countries_json_git_ref',
+                            help="A git ref from the everypolitician-data repo")
+
+    def handle(self, **options):
+        verbose_level = options['verbosity']
+
+        # FIXME: This SHA is currently pointing at a branch which has the
+        # `peoples_assembly` identifier, but this isn't on master because
+        # South Africa is broken in EveryPolitician at the moment.
+        #
+        # TODO: Make this SHA configurable.
```

Comment:
I think these comments can be removed now :)

Reply:
Oops, yep!